notNeededPackages.json

```diff
@@ -76,6 +76,12 @@
             "typingsPackageName": "redux",
             "sourceRepoURL": "https://github.com/reactjs/redux",
             "asOfVersion": "3.6.0"
+        },
+        {
+            "libraryName": "ajv",
+            "typingsPackageName": "ajv",
+            "sourceRepoURL": "https://github.com/epoberezkin/ajv",
+            "asOfVersion": "1.0.0"
```

Comment:
I'm afraid not related.
The version of @types/ajv is not related to the version of ajv.
The version of @types/ajv is 0.0.5, the `asOfVersion` should > 0.0.5, and should show a break change, so I think 1.0.0 should be OK.